src/code-examples/tabs/html.js

```diff
@@ -1,26 +1,25 @@
-const html = `
-    <div class="tabs">
-      <nav role="tablist" class="manual">
-        <button id="tab-1" type="button" role="tab" aria-selected="true" aria-controls="tabpanel-1">Section 1</button>
-        <button id="tab-2" type="button" role="tab" aria-selected="false" aria-controls="tabpanel-2" tabIndex="-1">Section 2</button>
-        <button id="tab-3" type="button" role="tab" aria-selected="false" aria-controls="tabpanel-3" tabIndex="-1">Section 3</button>
-      </nav>
+const html = `<div class="tabs">
+  <nav role="tablist" aria-labelledby="tablist-1" class="manual">
+    <button id="tab-1" type="button" role="tab" aria-selected="true" aria-controls="tabpanel-1">Section 1</button>
+    <button id="tab-2" type="button" role="tab" aria-selected="false" aria-controls="tabpanel-2" tabIndex="-1">Section 2</button>
+    <button id="tab-3" type="button" role="tab" aria-selected="false" aria-controls="tabpanel-3" tabIndex="-1">Section 3</button>
```

Comment:
We have some camelCased `tabIndex` attributes here that should be changed to `tabindex`.

Reply:
☝️ this is true! 🤦‍♀️ swapping between JSX and plain html will get you

Reply:
Oh jeeze- yup. It totally is from going back & forth. Here's the fix: https://github.com/sparkbox/accessible-components/pull/134/commits/fdc24e64505e5ce45cfd373ac47afc33f7608563